.circleci/config.yml

```diff
@@ -6315,12 +6315,6 @@ workflows:
           name: pytorch_linux_xenial_cuda10_1_cudnn7_py3_multigpu_test
           requires:
             - pytorch_linux_xenial_cuda10_1_cudnn7_py3_gcc7_build
-          filters:
-            branches:
-              only:
-                - master
-                - /ci-all\/.*/
-                - /release\/.*/
```

Comment:
What are the reason for these changes in the ci config?

Reply:
This is temporary change to force tests run for this PR. I'll undo it in next update to PR